packages/dbml-core/__tests__/importer/postgres_importer/input/general_schema.in.sql

```diff
@@ -102,3 +102,48 @@ COMMENT ON TABLE "users" IS 'User data';
 COMMENT ON TABLE "users" IS 'Store user data';
 
 COMMENT ON TABLE "products" IS 'Products table comment';
+
+-- issue 324
+CREATE TABLE "public"."accounts" (
+  "id" integer GENERATED ALWAYS AS IDENTITY NOT NULL,
+  "domain" character varying(100),
+  "name" character varying(100),
+  "slug" character varying(10),
+  CONSTRAINT "accounts_pkey" PRIMARY KEY ("id")
+) WITH (oids = false);
+
+-- issue 281
+CREATE TABLE public.tests ( id bigint NOT NULL, type character varying DEFAULT 'testing'::character varying );
+
+-- issue 427
+CREATE TABLE public.users2 (
+    username character varying NOT NULL,
+    hashed_password character varying NOT NULL,
+    full_name character varying NOT NULL,
+    email character varying NOT NULL,
+    password_changed_at timestamp with time zone DEFAULT '0001-01-01 00:00:00+00'::timestamp with time zone NOT NULL,
+    created_at timestamp with time zone DEFAULT now() NOT NULL
+);
+
+-- issue 248
+create table table1 (
+    field text not null,
+    field2 text not null,
+    primary key(field, field2)
+);
+
+create table if not exists table2 (
+    id int generated always as identity primary key,
+    field text,
+    field2 text,
+    foreign key (field, field2) references table1(field, field2)
+);
+
+-- issue 222
+CREATE TABLE test_table (
+	"name" varchar NOT NULL,
+	id varchar NULL,
+	linked_name varchar NOT NULL,
+	CONSTRAINT test_table_pk PRIMARY KEY ("name", id),
+	CONSTRAINT test_table_fk FOREIGN KEY (linked_name,id) REFERENCES test_table("name",id)
+);
```

Comment:
Please add the issue links inside the code nha em

Reply:
Added in https://github.com/holistics/dbml/pull/416/commits/5acb21aed42fee4aa1c5c3cc69a6d60a431920ab